tests/php/ORM/DataObjectTest.php

```diff
@@ -188,8 +188,8 @@ public function testConstructAcceptsValues()
         $this->assertEquals('John', $player->FirstName);
         $this->assertEquals('Doe', $player->Surname);
 
-        // IDs should be stored as integers, not strings
-        $player = new DataObjectTest\Player(array('ID' => '5'));
+        // Note that automatic conversion of IDs to integer no longer happens as the DB layer does that for us now
+        $player = new DataObjectTest\Player(array('ID' => 5));
```

Comment:
Feels a bit light on unit tests - we've gone from accepting two states to three states. Surely that should require another test case?